src/web/twig/Extension.php

```diff
@@ -944,11 +944,20 @@ public function shuffleFunction($arr)
      * @param bool|null $namespace Whether class names and IDs within the SVG
      * should be namespaced to avoid conflicts with other elements in the DOM.
      * By default the SVG will only be namespaced if an asset or markup is passed in.
-     * @param string|null $class A CSS class name that should be added to the `<svg>` element.
+     * @param array|string|null $attributes A list of attributes that should be added to the root `<svg>` element.
+     * @param array|null $elements Any elements that should be added under the  root`svg` element
+     * @param string|null $class Deprecated - use $attributes instead.
      * @return Markup|string
      */
-    public function svgFunction($svg, bool $sanitize = null, bool $namespace = null, string $class = null)
+    public function svgFunction($svg, bool $sanitize = null, bool $namespace = null, $attributes = [], array $elements = [], string $class = '')
     {
+        // Deprecate the $class argument. Allow both svg(class='class') and svg('', true, true, 'class')
+        $oldClass = is_string($attributes) ? $attributes : $class;
+        if ($oldClass) {
+            $attributes = ['class' => $oldClass];
+            Craft::$app->getDeprecator()->log('Extension::svgFunction()', 'Passing a string $class argument is deprecated. Pass an array with a `class` key instead.');
```

Comment:
The log key should be `'svg(class)'` – or something like that, to specifically refer to the `class` argument.

Reply:
![image](https://user-images.githubusercontent.com/6965875/63208710-ec82af80-c0d7-11e9-9b8b-43cbe18aa17b.png)